app/controllers/clearance/sessions_controller.rb

```diff
@@ -33,4 +40,8 @@ def url_after_create
   def url_after_destroy
     sign_in_url
   end
+
+  def url_for_signed_in_users
+    url_after_create
```

Comment:
Looks like there is already a [`UsersController#url_after_create`](https://github.com/thoughtbot/clearance/blob/master/app/controllers/clearance/users_controller.rb#L27). My mistake! Sorry about that.